sisl/geom/bilayer.py

```diff
@@ -104,7 +108,7 @@ def bilayer(bond=1.42, bottom_atom=None, top_atom=None, stacking='AB',
     if rep > 1:
         # Remove atoms outside cell
         cell_box = Cuboid(bilayer.cell)
-        cell_box.set_origo([-0.0001] * 3)
+        cell_box.set_center([-0.0001] * 3)
```

Comment:
I am not too happy about this. With origo all atoms are inside the unit cell, with center only 1/4 of the atoms are in the unit cell. This was why I changed it.

Also, with origo the atoms in the middle are on top, with center, the atoms at origo is on top of each other?
If the geometry looks better then the geometry should be shifted back.

Reply:
I just had a look between yours and mine. To be honest I don't really have a preference on the appearance. I'll shift everything to the primary unit cell and keep your implementation.